lib/elixir/lib/code.ex

```diff
@@ -629,10 +633,8 @@ defmodule Code do
   # unsupported chunk version
   defp lookup_docs(_, _), do: nil
 
-  @doc_sections [:docs, :moduledoc, :callback_docs, :type_docs]
-
   defp do_lookup_docs(docs, :all), do: docs
-  defp do_lookup_docs(docs, kind) when kind in @doc_sections,
+  defp do_lookup_docs(docs, kind) when kind in @doc_kinds,
```

Comment:
If `kind in @doc_kinds` moves to there is no need to have it here, but you need to add it to the second `def get_docs` definition as well. I think probably it doesn't worth it to move it then.